File: Javascript/factorial.js
function calculateFactorial(number) {
    if (number === 0 || number === 1) {
      return 1;
    } else {
      return number * calculateFactorial(number - 1);
    }
  }


  var num = 5;
  const result = calculateFactorial(num);
  console.log(`Factorial of ${num} is: ${result}`);

File: C/factorial.c
#include <stdio.h>
int factorial(int);
int main(){
    int n,result;
    printf("Enter a non negative number: ");
    scanf("%d",&n);
    result = factorial(n);
    printf("The factorial of %d is %d",n,result);
    return 0;
}
int factorial(int n){
    if(n>1){
        return n*factorial(n-1);
    }
    else{
        return 1;
    }
}